portfolio/src/main/webapp/index.html

```diff
@@ -14,5 +14,42 @@ <h1>My Portfolio</h1>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
     </div>
+
+    <button class="open-button" id = "myButton" onclick="openForm()">Open Form</button>
+    <div id="myForm" class="form">
+        <div class="form-popup" id="myForm">
```

Comment:
Suggest an id more informative than "myForm".

Reply:
I gave it a more informative name